index.html

```diff
@@ -31,7 +31,139 @@ <h1>I’m Mariane
      <li><a href=""><img src="images/vector2.svg" alt=""></a></li>
     </ul>
    </section>
+   <section>
+    <article class="snapshoot"><img src="images/Snapshoot.png" alt=""></article>
+    <article>
+     <h1>Tonic</h1>
+     <div class="frame">
+      <h2>CANOPY</h2>
+      <div></div>
+      <p >Back End Dev</p>
+      <div></div>
+      <p>2015</p>
+     </div>
+     <div class="text">
+      <p>A daily selection 
+       of privately personalized reads; no accounts or sign-ups required.</p>
+     </div>
+     <div class="lang">
+      <p>html</p>
+      <p>css</p>
+      <p>javaScript</p>
+     </div>
+     <button type="button">See Project</button>
+    </article>
+   </section>
+   <section>
+    <article class="snapshoot"><img src="images/Snapshoot1.png" alt=""></article>
+    <article>
+     <h1>Multi-Post Stories</h1>
+     <div class="frame">
+      <h2>CANOPY</h2>
+      <div></div>
+      <p >Back End Dev</p>
+      <div></div>
+      <p>2015</p>
+     </div>
+     <div class="text">
+      <p>A daily selection 
+       of privately personalized reads; no accounts or sign-ups required.</p>
+     </div>
+     <div class="lang">
+      <p>html</p>
+      <p>css</p>
+      <p>javaScript</p>
+     </div>
+     <button type="button">See Project</button>
+    </article>
+   </section>
+   <section>
+    <article class="snapshoot"><img src="images/Snapshoot2.png" alt=""></article>
+    <article>
+     <h1>Tonic</h1>
+     <div class="frame">
+      <h2>CANOPY</h2>
+      <div></div>
+      <p >Back End Dev</p>
+      <div></div>
+      <p>2015</p>
+     </div>
+     <div class="text">
+      <p>A daily selection 
+       of privately personalized reads; no accounts or sign-ups required.</p>
+     </div>
+     <div class="lang">
+      <p>html</p>
+      <p>css</p>
+      <p>javaScript</p>
+     </div>
+     <button type="button">See Project</button>
+    </article>
+   </section>
+   <section>
+    <article class="snapshoot"><img src="images/Snapshoot3.png" alt=""></article>
+    <article>
+     <h1>Multi-Post Stories</h1>
+     <div class="frame">
+      <h2 class="tc">CANOPY</h2>
+      <div></div>
+      <p >Back End Dev</p>
+      <div></div>
+      <p>2015</p>
+     </div>
+     <div class="text">
+      <p>A daily selection 
+       of privately personalized reads; no accounts or sign-ups required.</p>
+     </div>
+     <div class="lang">
+      <p>html</p>
+      <p>css</p>
+      <p>javaScript</p>
+     </div>
+     <button type="button">See Project</button>
+    </article>
+   </section>
   </main>
-  <footer></footer>
+  <footer>
+   <section class="Fabout">
+    <h1>About<br>Myself</h1>
```

Comment:
- [ ] As mentioned previously, you should consider making this an `<h2>` tag instead of `<h1>`.